packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -48,7 +57,14 @@ typedef void GestureDragCancelCallback();
 ///  * [PanGestureRecognizer], for drags that are not locked to a single axis.
 abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   /// Initialize the object.
-  DragGestureRecognizer({ Object debugOwner }) : super(debugOwner: debugOwner);
+  DragGestureRecognizer({
+    DragStartBehavior dragStartBehavior,
+    Object debugOwner,
+  })  : dragStartBehavior = dragStartBehavior ?? DragStartBehavior.start,
```

Comment:
defaults should be in the constructor arguments, then you can just assert != null

Reply:
The problem is the subclasses calling this constructor are always passing a (possibly null) value, so the default constructor argument is never picked up. I could add the default in each of the three subclasses, just thought this was a better option.

Reply:
Yeah, just update all the constructors.